storage/src/vespa/storage/distributor/distributor_operation_context.h

```diff
@@ -23,15 +23,14 @@ class DistributorOperationContext {
 public:
     virtual ~DistributorOperationContext() {}
     virtual api::Timestamp generate_unique_timestamp() = 0;
-    virtual BucketOwnership check_ownership_in_pending_and_current_state(const document::Bucket &bucket) const = 0;
     virtual void update_bucket_database(const document::Bucket& bucket,
                                         const BucketCopy& changed_node,
                                         uint32_t update_flags = 0) = 0;
     virtual void update_bucket_database(const document::Bucket& bucket,
                                         const std::vector<BucketCopy>& changed_nodes,
                                         uint32_t update_flags = 0) = 0;
     virtual void remove_node_from_bucket_database(const document::Bucket& bucket, uint16_t node_index) = 0;
-    virtual const DistributorBucketSpaceRepo& bucket_space_repo() const = 0;
```

Comment:
Better than making the check_ownership_in_pending_and_current_state const and the map mutable ?
Now you expose also the setXXX methods.